.prettierignore

```diff
@@ -12,10 +12,4 @@ examples/slack-clone/nextjs-slack-clone/full-schema.sql
 # ignore files with custom js formatting
 apps/docs/pages/guides/auth/*.mdx
 apps/docs/pages/guides/integrations/*.mdx
-studio/public
-studio/__mocks__
-studio/.next
-studio/.storybook
-studio/.swc
-studio/.turbo
-studio/node
+apps/studio/public
```

Comment:
Why are we keeping the public folder but removing all the other stuff?

Reply:
As you removed the file inside studio, should we add .swc, .turbo etc again?

Reply:
You're right, added `apps/**/.turbo` here. The other folders are probably a leftover from the previous git repo.